Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -261,6 +261,11 @@ void VideoConfig::VerifyValidity()
       iStereoMode = 0;
     }
   }
+  if (!backend_info.bSupportsBBox && g_ActiveConfig.bBBoxEnable)
+  {
+    PanicAlertT("Your graphics card does not support bounding box emulation this game requires. "
+                "You will experience bugs and crashes. This is not a Dolphin bug.");
```

Comment:
@Helios747 I think "which can lead to" would be better phrasing than "which leads to". Also it may make more sense to say "bugs or freezes" rather than "bugs and freezes". Otherwise looks better to me.